ref(angular): Update Angular SDK docs for `@sentry/angular-ivy` (#6356)
getsentry/sentry-docs: src/platform-includes/getting-started-install/javascript.angular.mdx

```bash {tabTitle:npm}
# Angular 12 and newer:
npm install --save @sentry/angular-ivy @sentry/tracing

# Angular 10 and 11:
npm install --save @sentry/angular @sentry/tracing
```

```bash {tabTitle:Yarn}
# Angular 12 and newer:
yarn add @sentry/angular-ivy @sentry/tracing

# Angular 10 and 11:
yarn add @sentry/angular @sentry/tracing
```

### Angular Version Compatibility

> Because of how Angular libraries are compiled, you need to use the correct version of the Sentry SDK for your Angular version:

Reviewer: ```suggestion
Because of the way Angular libraries are compiled, you need to use a specific version of the Sentry SDK for each corresponding version of Angular as shown below:
```

| Angular version | Recommended Sentry SDK              |
| --------------- | ----------------------------------- |
| 12 and newer    | `@sentry/angular-ivy`               |
| 10, 11          | `@sentry/angular`                   |
| 2 - 9\*         | `@sentry/angular@^6.x`              |
| 1 (AngularJS)\* | [AngularJS Integration](./angular1) |

<sup>

>   * Support for Angular versions below 10 was discontinued in version 7 of the SDK.
>   If you need to use Angular 9 or older you can try version 6 (<code>@sentry/angular@^6.x</code>) of the SDK.
>   Please note that this SDK version is no longer maintained or tested.

Reviewer: ```suggestion
  * Support for any Angular version below 10 was discontinued in version 7 of the SDK.
  If you need to use Angular 9 or older, try version 6 (<code>@sentry/angular@^6.x</code>) of the SDK.
  Note, that this version of the SDK is no longer maintained or tested.
```
